Dockerfile

```diff
@@ -1,17 +1,20 @@
-FROM python:3.8.3
+FROM python:3.8.3-buster AS build
 
-ARG PIP_ARGS
+COPY . /src
+WORKDIR /src
+RUN python3 setup.py bdist_wheel
```

Comment:
Is it safe to assume that this command runs python 3.8.3 and not the 3.7 default of debian buster?

Reply:
Considering it's the default python docker image named "3.8.3", then I would say "yes, it's safe to assume" :)
But, for clarity's sake, I ran python --version:

> docker run --rm python:3.8.3-buster python --version
Python 3.8.3